test/e2e/e2e_test.go

```diff
@@ -0,0 +1,33 @@
+//+build e2e
+
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"flag"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
```

Comment:
validate-imports should have complained about this; I think it is because it is not checking /test - please can you update the Makefile?